src/TConsole.h

```diff
@@ -79,9 +79,9 @@ class TConsole : public QWidget
     Q_DECLARE_FLAGS(ConsoleType, ConsoleTypeFlag)
 
     enum ControlCharacterMode {
-        NoControlCharacterReplacement = 0x0,
-        PictureControlCharacterReplacement = 0x1,
-        OEMFontControlCharacterReplacement = 0x2
+        AsIs = 0x0,
+        Picture = 0x1,
+        OEM = 0x2
     };
     Q_ENUM(ControlCharacterMode)
```

Comment:
I think that @keneanung pointed out , and in hindsight, I agree with him in https://github.com/Mudlet/Mudlet/pull/6017#issuecomment-1078193982 - the outer `TConsole` scope is not that helpful - so pulling this out of the `TConsole` class definition so one uses the `ControlCharacterMode` (or better IMHO) `ControlCharacterReplament` scope works better when using IDEs that auto-suggest completions... as `TConsole::AsIs` is not going to help anyone to understand what it is for..